Makefile

```diff
@@ -12,6 +12,8 @@ endif
 
 ifeq ($(shell uname -s),Darwin)
 	export CGO_LDFLAGS=-framework Foundation -framework SystemConfiguration
```

Comment:
So `-ldl` is dynamic linking library and `-lm` is math library. Both are not needed for Mac OS because the OS use different linking mechanism and math is part of C stdlib anyway. But we should also check requirment for `-lm` on other platforms though

Reply:
I'll add it as full git message